nupm/mod.nu

```diff
@@ -0,0 +1,16 @@
+use utils/dirs.nu [ DEFAULT_NUPM_HOME nupm-home-prompt ]
+
+export-env {
+    # Ensure that $env.NUPM_HOME is always set when running nupm. Any missing
+    # $env.NUPM_HOME during nupm execution is a bug.
+    if 'NUPM_HOME' not-in $env {
+        $env.'NUPM_HOME' = $DEFAULT_NUPM_HOME
+    }
+}
+
+# Nushell Package Manager
+export def main [] {
+    nupm-home-prompt
```

Comment:
a question about that though :thinking: 

- the `NUPM_HOME` variable is always set thanks to the `export-env` block above :+1: 
- there is a check when running `nupm` on its own
- but what about running `nupm install`, will the Nupm directory have the same kind of checks? :thinking:

Reply:
Eventually, every `nupm` subcommand (that needs it) will have the NUPM_HOME check, I just wanted to keep this PR small, I'll do another one for `nupm install`.

Reply:
ooh ok gotcha :+1: